### ISSUE TITLE
security not parsing "none"

### ISSUE THREAD

**Issue by Daxten** [swagger-api/swagger-parser#492]
see https://github.com/OAI/OpenAPI-Specification/issues/14#issuecomment-297457320

empty security object is not getting parsed
```
"security": [
  {},
  {"oauth": […]},
  {"token-1": []},
  {"token-2": []}
]
```
parsed has only 3 elements

**Comment by gracekarina** [comment]
Hi @Daxten, Thanks for the report, please checkout if this fixes the issue in branch 2.0

**Comment by gracekarina** [comment]
Hi, @Daxten  please let us know if it's still an issue
